smart_proxy_acd.gemspec

```diff
@@ -14,4 +14,5 @@ Gem::Specification.new do |s|
   s.license = 'GPLv3'
 
   s.add_development_dependency('rubocop', '0.50.0')
+  s.add_runtime_dependency('smart_proxy_dynflow', '>= 0.5.0')
```

Comment:
Should it also depend on REX as the previous core did or is that no longer needed?

Reply:
Not sure why it was there in the first place, from what I've seen it didn't seem to be using anything from rex core